### PR TITLE
Add result

### PR DESCRIPTION
**Pull request by Supernova-45**
**What I tried:** 
- Used torch.compile() and torch.set_float32_matmul_precision("high") 
- Used bf16 autocast for training and validation
- Used squared ReLU as the FFN instead of SwiGLU or SiLU 
- Tied input embeddings and LM head weights 
- Experimented with Muon on attention matrices + Adam on embeddings / LM head but ultimately used AdamW for all optimizations 
- Tuned hyperparameters: 
`"d_model": 768, "num_heads": 12, "num_layers": 6, "d_ff": 3072, "lr": 0.003, "lr_min": 1e-4, "warmup_steps": 500, "betas": [0.9, 0.999], "eps": 1e-8, "weight_decay": 0.1, "num_steps": 30000, "batch_size": 128, "max_grad_norm": 1.0,`

Final validation loss: 3.251
[](https://api.wandb.ai/links/alexandrasuriya-ml/1nybxa71)

### CODE REVIEW

**Comment by hermabr** [comment]
Please use your val loss of 3.26624
